packages/components/forms/src/Checkbox/README.mdx

```diff
@@ -9,12 +9,19 @@ storybook: 'https://v4-f36-storybook.netlify.app/?path=/story/form-elements-chec
 typescript: ./Checkbox.tsx,./CheckboxGroup.tsx
 ---
 
-`Checkbox` is a form component that shows a checkbox with it's label.
-`Checkbox.Group` is a component that makes it easier to handle group of checkboxes.
+`Checkbox` is a form component that shows a checkbox with it's label. `Checkbox.Group` is a component that makes it easier to handle group of checkboxes.
 
-## How to use Checkbox
+If you want to allow a user to select only one out of multiple optionsIf you want to let a user select multiple options, use the [Radio](/components/radio) component.
 
-You can use the Checkbox as a controlled or uncontrolled input.
+## Import
+
+```js static=true
+import { Checkbox } from '@contentful/f36-components';
+// or
+import { Checkbox } from '@contentful/f36-forms';
+```
+
+## Examples
 
 ### Using as a controlled input
```

Comment:
could you please copy the beginning of `controlled` and `uncontrolled` sections from Radio button? there's a little explanation what the difference.